shell/src/main/java/alluxio/cli/fs/command/RmCommand.java

```diff
@@ -63,6 +63,13 @@ public final class RmCommand extends AbstractFileSystemCommand {
           .hasArg(false)
           .desc("remove data and metadata from Alluxio space only")
           .build();
+  private static final Option DELETE_MOUNT_POINT =
+      Option.builder()
```

Comment:
```suggestion
      Option.builder("d")
```

Reply:
Let's use `-d` as a shorthand for `--deleteMountPoint`? Or `-m`? WDYT @dbw9580

Reply:
Use -m as short option name